Havit.Blazor.Components.Web.ECharts/HxEChart.razor.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.JSInterop;
+
+namespace Havit.Blazor.Components.Web.ECharts;
+
+/// <summary>
+/// Component for convenient rendering of Apache ECharts.
+/// </summary>
+public partial class HxEChart : IAsyncDisposable
+{
+	private static readonly JsonSerializerOptions s_JsonSerializerOptions;
+
+	/// <summary>
+	/// Unique identifier for the HTML element representing the chart.
+	/// </summary>
+	[Parameter] public string ChartId { get; set; } = $"hx-echart-{Guid.NewGuid()}";
```

Comment:
Is it worth exposing this Parameter at this time?

The component does not properly handle the case when ChartId changes - when I tested it the auto resize functionality in the basic demo broke...

Reply:
You don't want to change the value, you want to be able to find the instance of ECharts in DOM to be able to do some manual adjustments/interactions.
I find this "safe enough" :-D